examples/feature_demo/video_yuv.py

```diff
@@ -0,0 +1,176 @@
+"""
+Video yuv
+=========
+
+This example demonstrates how to show video data in a very efficient way,
+eliminating as much data-copies and color conversion operations as we can.
+
+Reading
+-------
+
+Many videos are encoded using yuv420p. Therefore, loading the video in
+that format, instead of requesting rgb frames, saves computation and
+data copies upon reading.
+
+Uploading
+---------
+
+The YUV data can also be uploaded directly to the texture, albeit in
+three chunks. RGBA frames can be send in one chunk (although it is
+considerably larger). For RGB data, the data must first be converted
+to RGBA, which is costly. (This packing is normally done automatically
+by the pygfx Texture).
+
+Size
+----
+
+Frames in yuv420p format are half the size of RGB frames (and even less
+compared to RGBA) so the upload itself is cheaper, and memory
+consumption on the CPU and GPU are reduced.
+
+Benchmarks
+----------
+
+This script can be done to benchmark for different formats. We've found
+that using yuv420p data is certainly the fastest. Using rgba is a bit
+slower, although it consumes more memory, so its disadvantage may
+increase for heavy workloads. Reading the data as RGB has terrible
+performance. This is unfortunate, because it's the format that many
```

Comment:
```suggestion
performance. We believe that this is caused by the fact that RGB images must
be converted to RGBA pior to uploading them to the GPU.
This is unfortunate, because it's the format that many
```